tests/integration/test_utils/data_source_config/snowflake.py

```diff
@@ -48,29 +48,30 @@ class SnowflakeConnectionConfig(BaseSettings):
     SNOWFLAKE_PW: str
     SNOWFLAKE_ACCOUNT: str
     SNOWFLAKE_DATABASE: str
-    SNOWFLAKE_SCHEMA: str
     SNOWFLAKE_WAREHOUSE: str
     SNOWFLAKE_ROLE: str = "PUBLIC"
 
     @property
     def connection_string(self) -> str:
         return (
             f"snowflake://{self.SNOWFLAKE_USER}:{self.SNOWFLAKE_PW}"
-            f"@{self.SNOWFLAKE_ACCOUNT}/{self.SNOWFLAKE_DATABASE}/{self.SNOWFLAKE_SCHEMA}"
+            f"@{self.SNOWFLAKE_ACCOUNT}/{self.SNOWFLAKE_DATABASE}"
             f"?warehouse={self.SNOWFLAKE_WAREHOUSE}&role={self.SNOWFLAKE_ROLE}"
         )
 
 
 class SnowflakeBatchTestSetup(SQLBatchTestSetup[SnowflakeDatasourceTestConfig]):
-    @override
     @property
+    @override
     def connection_string(self) -> str:
+        schema = self.schema
+        assert schema
```

Comment:
Why do you need to assert the schema here? It doesn't look like it is returned in the connection string on the line below.